tutorials/README.md

```diff
@@ -16,47 +16,81 @@ Check that your tutorial renders properly.
-(that is, the URL after `https://learning-api.quantum.ibm.com/admin/content/`).
-The copied URL should be of the form `tutorials/<UUID>`. Next, make a new entry
-in the `learning-api.conf.yaml` file in this folder. Set the `path` attribute
-to the name of the tutorial folder in this repo, and `urlProduction` to the
-tutorial ID you copied earlier. Repeat the same process for the staging
-environment (https://learning-api-dev.quantum.ibm.com/admin/).
+1. **Create a new page on the learning platform**
 
-Next, you will need to add the tutorial to our API token permissions. To do
-that, contact Abdón Rodríguez and ask to add the tutorial IDs to the
-Qiskit/documentation API keys.
+   For both environments (staging and production), log on to the CMS and go to
+   the "tutorials" tab. Click "Create item" (the blue `+` in the top-right
+   corner) and enter all the information, but leave the content field blank.
+   This will create the tutorial in "draft" mode.
 
-To push content from this repo to the learning platform, run the [Deploy
-tutorials](https://github.com/Qiskit/documentation/actions/workflows/deploy-tutorials.yml)
-workflow. This will push the content of **all** tutorials from the main branch.
-Once your content is uploaded, you can go back to the admin panel for that
-tutorial and publish the lesson to take it out of draft mode.
+2. **Give our token permission to push to the tutorial**
+
+   > TODO: Improve this; I don't have permission to do this so can't give more details.
+
+   Next, we need to give the token in this repo permission to push to the new
+   tutorial. In the CMS, go to "Users", then give "Token: Qiskit/documentation"
+   permission to push to the new tutorial.
+
+3. **Link the notebook in this repo to the learning platform**
+
+   For our action to upload content from this repo, we need to link the
+   tutorial you created in the learning platform to the file in this
+   repository. Repeat this for staging and production.
+
+   1. **Get the URL**
+
+      Navigate to the tutorial in the CMS and copy the URL relative to
+      `content` (that is, the URL after
+      `https://learning-api.quantum.ibm.com/admin/content/`). The copied URL
+      should be of the form `tutorials/a00eed3f-fdba-416d-8f22-7a2522de02bc`.
+
+   2. **Update `tutorials/learning-api.conf.yaml`**
+
+      Next, make a new entry in the `learning-api.conf.yaml` file in this
+      folder. Set the `path` attribute to the name of the tutorial folder in
+      this repo, and the appropriate URL to the URL you copied in the previous
+      sub-step.
+
+4. **Push the content from this repo to the learning platform**
+
+   To push content from this repo to the learning platform, run the [Deploy
+   tutorials](https://github.com/Qiskit/documentation/actions/workflows/deploy-tutorials.yml)
+   workflow. This will push the content of **all** tutorials from the main
+   branch. Do this for both staging and production.
+
+5. **Review and publish**
```

Comment:
I think we should add an additional step for production - we need to remind people to set the correct access controls for network only content